docs/how-xtrabackup-works.md

```diff
@@ -1,123 +1,149 @@
+
+The `BACKUP_ADMIN` privilege is required to execute the `LOCK INSTANCE FOR BACKUP` statement. This privilege manages database backups, ensures data consistency, and performs backup-related tasks.
+
+### Locking the binary log
+
+The `performance_schema.log_status` table in MySQL is a valuable resource for backups. This table provides critical information that 
+allows a backup tool to copy necessary log files without locking the resources for the duration of the copy process. When queried, 
+the `log_status` table causes the server to momentarily block logging and related administrative changes to populate the table 
+with data, and then the server releases the resources. This table identifies the point in the 
+source's binary log and `gtid_executed` record the backup should copy, including the relay log for each replication channel. 
+Additionally, it offers essential details, such as the last log sequence number (LSN) and the 
+LSN of the previous checkpoint for InnoDB.
+
+To use the `log_status` table effectively, you must have the `BACKUP_ADMIN` privilege and the `SELECT` privilege.
+
+The table contains several columns, including `SERVER_UUID`, which is the server's unique identifier, and `LOCAL`, which provides the log 
+position state information from the source as a JSON object. This table includes the current binary log file name and position, 
+as well as the `gtid_executed` value at the time the table was accessed. The `REPLICATION` section is a JSON array containing detailed information about each replication channel, such as the channel name, current relay log file, and position. The `STORAGE_ENGINES` section provides 
+JSON objects with keys for each applicable storage engine, detailing their specific log information.
+
+To perform a backup using the `log_status` table, one would typically follow these steps:
+1. Ensure you have the necessary privileges to access the `log_status` table.
+2. Query the `log_status` table to retrieve the current logging state.
+3. Use the information from the `LOCAL` and `REPLICATION` sections to determine the log files and positions that need to be copied.
+4. Copy the log files up to the specified positions without the need to lock the resources for an extended period.
```

Comment:
PXB doesn't copy binlog files or relay log files. we only store the current binlog file info into an index file in backup.

server ensured that by locking all engines binlog, innodb, the data copied until a LSN corresponds to specific binlog/gtid position.

But we do use the LSN information from PFS log_status to stop copying the redo thread at the LSN provided by PFS.